packages/events/src/util.ts

```diff
@@ -55,3 +56,48 @@ export function getEventServiceUrl(
     }
   });
 }
+
+/**
+ * Get the events query based on type.
+ * @param type - string to indicate event type with the options `upcoming`, `past`, `cancelled` and `draft`
+ * @param params - query features request options
+ * @param requestOptions - request options that may include authentication
```

Comment:
I think this should just take `type: string; requestOptions: IQueryFeaturesRequestOptions`. There's no need for `params`. `requestOptions` would have the `where` and `orderByFields` either as it's own properties, or under it's own `params`.